src/Context/RestContext.php

```diff
@@ -83,7 +83,7 @@ public function theResponseShouldBeEqualTo(PyStringNode $expected)
     {
         $expected = str_replace('\\"', '"', $expected);
         $actual   = $this->request->getContent();
-        $message = "The string '$expected' is not equal to the response of the current page";
+        $message = "Response is: $actual";
```

Comment:
Now we don't see the $expected anymore in the message.
Maybe something like `"The response of the current page is not '$expected', it is: $actual"`

Reply:
You see expected always right before the error message.

Reply:
That depends on your output formt. With the format (-f) `pretty` (which is default) you see the expected before the error message, but not with e.g. `progress` or `PhpStormBehatFormatter`.

Reply:
What about the following methods?

- [theHeaderShouldBeEqualTo](https://github.com/razbakov/contexts/blob/master/src/Context/RestContext.php#L111)
- [theHeaderShouldNotBeEqualTo](https://github.com/razbakov/contexts/blob/master/src/Context/RestContext.php#L124)
- etc.

It make more sense to see `$actual` in error message, rather than `$expected`

Reply:
Yes, the `$actual` should be in the message. But in my opinion, also the `$expected` to make life easier when something fails.